TeamcityRestApi/TeamcityConnector.fs

```diff
@@ -825,6 +828,18 @@ type TeamcityConnector(httpconnector : IHttpTeamcityConnector) =
             let payload = sprintf """<builds><build id="%s"/></builds>""" buildid
             let requestDAta = httpconnector.HttpPutXmlContent(conf, url, payload)
             requestDAta.IsSuccessful
+        
+        member this.GetBranchesFromTeamcityConfiguration(conf: ITeamcityConfiguration, buildTypeId: string) =
+            let url = sprintf "app/rest/buildTypes/id:%s/branches" buildTypeId
+            let content = httpconnector.HttpRequest(conf, url, RestSharp.Method.Get).Content
```

Comment:
you can use type providers directly to access the data.. see how the other fucntions are done